erc831/src/main/kotlin/org/kethereum/erc831/ERC831.kt

```diff
@@ -1,9 +1,10 @@
 package org.kethereum.erc831
 
-// as defined in http://eips.ethereum.org/EIPS/eip-831
-
+/**
+ * as defined in [EIP-831](https://eips.ethereum.org/EIPS/eip-831)
```

Comment:
do I understand that 831 is not yet approved ERC, but still EIP?

Reply:
yea - still need to push this through the EIP process